packages/mobile/ios/Podfile.lock

```diff
@@ -1068,7 +1068,7 @@ SPEC CHECKSUMS:
   Flipper-RSocket: d9d9ade67cbecf6ac10730304bf5607266dd2541
   FlipperKit: d8d346844eca5d9120c17d441a2f38596e8ed2b9
   fmt: ff9d55029c625d3757ed641535fd4a75fedc7ce9
-  glog: 5337263514dd6f09803962437687240c5dc39aa4
+  glog: 40a13f7840415b9a77023fbcae0f1e6f43192af3
```

Comment:
Once `https://github.com/valora-inc/wallet/pull/2035` lands this shouldn't be necessary anymore.
I had to `rm -rf Pods` locally to get the right `glog` checksum which matched CI.